xml/System.Runtime.InteropServices/Marshal.xml

```diff
@@ -303,6 +303,9 @@
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
+> [!IMPORTANT]
+> This native memory allocator is a legacy API that should be used exclusively when called for by specific Win32 APIs on the Windows platform. On non-Windows platforms, use the <xref:System.Runtime.InteropServices.NativeMemory> class to allocate native memory.
```

Comment:
This is incomplete guidance. What should people do on Windows platforms or when they target TFM that does not have `S.R.I.NativeMemory`?

Reply:
I appreciate what you're getting at, but I don't think the doc will matter in practice all that much for those cases (TFM < .NET 6). Perhaps all that is needed in this case then is to say:

```
This native memory allocator is a legacy API that should be used exclusively when called for by specific Win32 APIs on the Windows platform or when targeting .NET versions earlier than .NET 6.0. On non-Windows platforms, use the <xref:System.Runtime.InteropServices.NativeMemory> class to allocate native memory.
```